packages/react-components/react-rating-preview/src/components/RatingItem/useRatingItem.tsx

```diff
@@ -81,6 +81,7 @@ export const useRatingItem_unstable = (props: RatingItemProps, ref: React.Ref<HT
         name: context.name,
         value: value - 0.5,
         checked: ratingValue === value - 0.5,
+        'aria-label': context.itemLabel ? context.itemLabel(value - 0.5) : `${value - 0.5}`,
```

Comment:
nit: to make things simpler, destructure `itemLabel` from context above, and give it a default value that just returns the number as a string:
```ts
  const { itemLabel = value => value + '' } = context;
```

Then this can be:
```suggestion
        'aria-label': itemLabel(value - 0.5),
```

And the fullValueInput:

```ts
        'aria-label': itemLabel(value),
```